homeassistant/components/binary_sensor/danfoss_air.py

```diff
@@ -0,0 +1,60 @@
+"""
+Binary sensors for Danfoss Air HRV.
+
+Configuration:
+    danfoss_air:
+        host: IP_OF_CCM_MODULE
```

Comment:
This doesn't make sense to have here, since it's not a configuration applied to the platform, but rather to the component.